chain/gen/genesis/genesis.go

```diff
@@ -228,6 +228,17 @@ func MakeInitialStateTree(ctx context.Context, bs bstore.Blockstore, template ge
 		}
 	}
 
+	// Create ipc gateway actor
+
+	gatewayAct, err := SetupIPCGateway(ctx, bs, av, "TestGateway", 10)
+	if err != nil {
+		return nil, nil, xerrors.Errorf("setup ipc gateway actor: %w", err)
+	}
+	gatewayAddress := "TODO: how to obtain this address?"
```

Comment:
If you do `gatewayAct.address` you get the address. Why do you need it? 

BTW, leaving here what we need to do to deploy the subnet-actor:
- We need to call the init actor to spawn an new instantiation of the actor:  https://github.com/consensus-shipyard/lotus/blob/a5b6896a7a4c830de910466f9feac3b95328cd6e/chain/actors/builtin/paych/message10.go#L21

Also, we should consider if we want to make the gateway actor a singleton actor or not. We deploy ours in genesis, but others could deploy their own instance of the gatway (I think not having it as singleton matches best the situation with user-defined actors).